tensorly/contrib/sparse/kruskal_tensor.py

```diff
@@ -1,5 +1,54 @@
-from ...kruskal_tensor import kruskal_to_tensor,unfolding_dot_khatri_rao
+from ...kruskal_tensor import kruskal_to_tensor, unfolding_dot_khatri_rao, kruskal_norm
 from .core import wrap
+from .backend import sparse_context
+
+from numpy import zeros, multiply
+import tensorly as tl
+
 
 kruskal_to_tensor = wrap(kruskal_to_tensor)
 unfolding_dot_khatri_rao = wrap(unfolding_dot_khatri_rao)
+
+
+def sparse_mttkrp(tensor, factors, n, rank, dims):
+    with sparse_context():
```

Comment:
I don't know if I need to have a `with sparse_context():` for every method in this file. The reason I include this line is because I call the `values` and `indices` methods inside this method, and the implementations for those only exist if the backend is sparse.